sources/packages/web/src/views/institution/InstitutionUserProfile.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <Message severity="info">
+    Please notice that the read-only information below is retrieved from your
+    BCeID account and it is not possible to change it here. If any read-only
+    information needs to be changed please visit
+    <a rel="noopener" href="https://www.bceid.ca/" target="_blank">bceid.ca</a>.
+  </Message>
+  <Card class="p-m-4">
+    <template #content>
+      <formio
+        formName="institutionUserProfile"
+        :data="initialData"
+        @submitted="submitted"
+      ></formio>
+    </template>
+  </Card>
+</template>
+
+<script lang="ts">
+import { ref, onMounted } from "vue";
+import { useRouter } from "vue-router";
+import formio from "../../components/generic/formio.vue";
+import { UserService } from "../../services/UserService";
+import { useToastMessage } from "@/composables";
+import {
+  InstitutionUserDetailsDto,
+  InstitutionUserPersistDto,
+} from "../../types";
+import { InstitutionRoutesConst } from "../../constants/routes/RouteConstants";
+
+export default {
+  components: { formio },
+  setup() {
+    // Hooks
+    const toast = useToastMessage();
+    const router = useRouter();
+    // Data-bind
+    const initialData = ref({});
+
+    const submitted = async (data: InstitutionUserDetailsDto) => {
+      const institutionUserPersistDto: InstitutionUserPersistDto = {
+        userEmail: data.userEmail,
+      };
+      try {
+        await UserService.shared.updateInstitutionUser(
+          institutionUserPersistDto,
+        );
+        toast.success("Updated!", "Institution and User successfully updated!");
```

Comment:
The message is mentioning the institution, but I believe that only the user is being updated here, right?

Reply:
Again a copy paste issue.  will fix it.

Reply:
Fixed